packages/xdl/src/Analytics.ts

```diff
@@ -55,10 +59,10 @@ export class AnalyticsClient {
   }
 
   private ensureUserIdentified() {
```

Comment:
either
- move content of this function to setUserProperties
- pass userId and userTraits as argument (and maybe rename this function to identify or identifyUser)